<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameOver</class>
 <widget class="QDialog" name="gameOver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>380</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Montserrat</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Game Over</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imglib.qrc">
    <normaloff>:/images/zorkIcon.png</normaloff>:/images/zorkIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(79, 87, 88);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="mainMenuButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Main Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leaderboardButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Leaderboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playAgainButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Play Again</string>
   </property>
  </widget>
  <widget class="QLabel" name="winLoseLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>250</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Caught!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="winLoseHeading">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>350</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>You failed to make it out in time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="winLoseText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>351</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>Tough luck. The kidnapper has found you after trying to escape and it is now likely that you will never make it out alive. T'was nice knowing ya.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Your time :</string>
   </property>
  </widget>
  <widget class="QLabel" name="bestLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Best time :</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerTime">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bestTime">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imglib.qrc"/>
 </resources>
 <connections/>
</ui>
